lib/chitragupta/util.js

```diff
@@ -10,6 +10,8 @@ function populateServerData(dataParam) {
   data.data.request = data.data.request || {};
   data.data.response = data.data.response || {};
 
+  data.log.uuid = cls.get("uuid") || {};
```

Comment:
Let's return a null or empty string rather than empty json as this is a string field.